sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -97,6 +97,28 @@ public abstract class RecordedTestBase : ClientTestBase
         /// </summary>
         public List<(string Header, string QueryParameter)> SanitizedQueryParametersInHeaders { get; } = new();
 
+        /// <summary>
+        /// Flag you can (temporarily) enable to save failed test recordings
+        /// and debug/re-run at the point of failure without re-running
+        /// potentially lengthy live tests.  This should never be checked in
+        /// and will throw an exception from CI builds to help make that easier
+        /// to spot.
+        /// The list of JSON path sanitizers to use when sanitizing a JSON request or response body.
```

Comment:
Is this last comment relevant to this setting? If so, could it be clarified?

Reply:
Woops will remove.